lib/src/dartdoc_options.dart

```diff
@@ -92,12 +90,10 @@ class CategoryConfiguration {
     var newCategoryDefinitions = <String, CategoryDefinition>{};
     for (var entry in yamlMap.entries) {
       var name = entry.key.toString();
-      String displayName;
-      String documentationMarkdown;
       var categoryMap = entry.value;
       if (categoryMap is Map) {
-        displayName = categoryMap['displayName']?.toString();
-        documentationMarkdown = categoryMap['markdown']?.toString();
+        var displayName = categoryMap['displayName']?.toString();
```

Comment:
haha easy win

Reply:
Yeah, my early Dart code was a little cringey, I'm happy to give it a touch up as part of this migration :-)